TYPO3.Flow/Classes/TYPO3/Flow/Persistence/Doctrine/Mapping/Driver/FlowAnnotationDriver.php

```diff
@@ -732,7 +737,13 @@ protected function evaluatePropertyAnnotations(ClassMetadataInfo $metadata)
                             break;
                         default:
                             if (strpos($propertyMetaData['type'], '\\') !== false) {
-                                if ($this->reflectionService->isClassAnnotatedWith($propertyMetaData['type'], Flow\ValueObject::class)) {
+                                if ($valueObjectAnnotation = $this->reflectionService->getClassAnnotation($propertyMetaData['type'], Flow\ValueObject::class)) {
```

Comment:
Ok, just checked again, I get following error during warmup:

```
Failure while evaluating property annotations for class "TYPO3\Flow\Tests\Functi
onal\Persistence\Fixtures\TestEntity": Property "embeddedValueObject" in "TYPO3\
Flow\Tests\Functional\Persistence\Fixtures\TestEntity" was already declared, but
 it must be declared only once
Type: Doctrine\ORM\Mapping\MappingException
  Code: 1382003497
  File:
...Data\Temporary\Testing\Cache\Code\Flow_Object_Classes\TYPO3_Flow_Persistence_Doctrine_Mapping_Driver_FlowAnnotationDriver.php
  Line: 414
```

maybe you can verify.